version.php

```diff
@@ -27,7 +27,7 @@
 defined('MOODLE_INTERNAL') || die();
 
 
-$plugin->version   = 2019100300;        // The current plugin version (Date: YYYYMMDDXX)
+$plugin->version   = 2019101600;        // The current plugin version (Date: YYYYMMDDXX)
 $plugin->release   = 2018070200;        // The current plugin version (Date: YYYYMMDDXX)
```

Comment:
please update this and always keep it in sync with the version

Reply:
Done in latest update